### PR TITLE
Update to Scala 3 syntax

### PR DESCRIPTION
**Pull request by vincenzobaz**

### CODE REVIEW

File: CheatSheet.md [hunk 1]
```diff
@@ -155,7 +153,8 @@ or
 
 - Traits are similar to Java interfaces, except they can have non-abstract members:
 ```scala
-        trait Planar { ... }
+        trait Planar:
+	  ...
```
Comment (julienrf) [inline CheatSheet.md]:
```suggestion
	   ...
```

File: CheatSheet.md [hunk 2]
```diff
@@ -273,16 +271,18 @@ can be written as follows:
 
 Pattern matches are also used quite often in anonymous functions:
 ```scala
-    val pairs: List[(Char, Int)] = ('a', 2) :: ('b', 3) :: Nil
+    val pairs: List[Option[Char]] = Some('a') :: None :: Some('b') :: Nil
```
Comment (julienrf) [inline CheatSheet.md]: This is not a list of pairs anymore

File: ScalaStyleGuide.md [hunk 3]
```diff
@@ -151,33 +150,32 @@ Since this is a course on functional programming, we want you to get used to wri
...
 
 
 ### #12 Eliminate redundant "If" Expressions
 
 Instead of
 
-    if (cond) true else false
+    if cond then true else false
 
 you can simply write
```
Comment (julienrf) [inline ScalaStyleGuide.md:181]: maybe add something about `end` markers in this file?

File: CheatSheet.md (hunk 2 above)
Comment (vincenzobaz) [reply CheatSheet.md]: Good catch! I thought that `Option` was better to show anonymous pattern match, as it is not required anymore for tuples

Comment (julienrf) [reply CheatSheet.md]: Oh, I understand now. Maybe just rename `pairs` to `options`?

**Review by julienrf — APPROVED** [review]

**Comment by vincenzobaz** [comment]
I do not have the accesses to merge this PR.
Please use the Github button to squash all the commits or I can do it manually

**Pull request merged**